plutus-tx-plugin/test/Blueprint/Tests/Lib.hs

```diff
@@ -49,27 +51,33 @@ newtype Bytes (phantom :: Type) = MkAcmeBytes BuiltinByteString
   deriving anyclass (AsDefinitionId)
 
 instance HasSchema (Bytes phantom) ts where
-  schema = SchemaBytes emptySchemaInfo{title = Just "SchemaBytes"} emptyBytesSchema
+  schema = SchemaBytes emptySchemaInfo { title = Just "SchemaBytes" } emptyBytesSchema
 
+{-# ANN type DatumPayload (SchemaComment "MkDatumPayload") #-}
```

Comment:
I think the name of this module is a bit confusing. We usually say "test lib" when we mean a module that provides helper functions for writing tests. E.g., we have `plutus-core-testlib`, `plutus-ledger-api-testlib` and `plutus-tx-testlib`. But this is just a specific test case.

On the other hand, things such as `goldenBlueprint` should belong to a testlib package.

Reply:
I have extracted `goldenJson` function out of the `goldenBlueprint`  and moved it to the `Tests/Lib` module, where it gets higher chance of re-use.